src/Form/WebPageArchiveForm.php

```diff
@@ -70,43 +78,19 @@ public function save(array $form, FormStateInterface $form_state) {
     $web_page_archive = $this->entity;
     $status = $web_page_archive->save();
 
-    if ($status) {
-      drupal_set_message($this->t('The %label web page archive was saved.', [
-        '%label' => $web_page_archive->label(),
-      ]));
+    switch ($status) {
+      case SAVED_NEW:
+        drupal_set_message($this->t('Created the %label Web page archive entity.', [
+          '%label' => $web_page_archive->label(),
+        ]));
+        break;
+
+      default:
+        drupal_set_message($this->t('Saved the %label Web page archive entity.', [
+          '%label' => $web_page_archive->label(),
+        ]));
```

Comment:
This _feels_ a little naive?  If it doesn't save for whatever reason, you'll see the "saved..." message regardless?